src/astronomer/flask_appbuilder/security.py

```diff
@@ -230,6 +230,26 @@ def manage_user_roles(self, user, roles):
         for role in desired:
             user.roles.append(self.find_role(role))
 
+    try:
+        # Python binding/closures are acting weird. Without this, the useage in `has_access` below can't
+        # find permissions!
+        global permissions
```

Comment:
`global` always makes me feel _iffy_ 😄

Reply:
Yeah, this was an _odd_ one, hence the comment. But without this it can't find the `permissions` variable.